view/src/pages/home.js

```diff
@@ -166,20 +276,92 @@ class Home extends Component {
             </div>
             <Divider />
             <List>
-              <ListItem button key="Todo" onClick={this.loadTodoPage}>
+              <ListItem
+                button
+                className={this.isSelected("farms") && classes.selected}
+                key="Farms"
+                onClick={this.loadFarmsPage}
+              >
                 <ListItemIcon>
-                  <NotesIcon />
+                  <NatureIcon
+                    className={this.isSelected("farms") && classes.selectedIcon}
+                  />
                 </ListItemIcon>
-                <ListItemText primary="Todo" />
+                <ListItemText primary="Farms" />
               </ListItem>
-
-              <ListItem button key="Account" onClick={this.loadAccountPage}>
+              <ListItem
+                className={this.isSelected("foodbanks") && classes.selected}
+                button
+                key="Food Banks"
+                onClick={this.loadFoodBanksPage}
+              >
+                <ListItemIcon>
+                  <BankIcon
+                    className={
+                      this.isSelected("foodbanks") && classes.selectedIcon
+                    }
+                  />
+                </ListItemIcon>
+                <ListItemText primary="Food Banks" />
+              </ListItem>
+              <ListItem
+                button
+                className={this.isSelected("deals") && classes.selected}
+                key="Deals"
+                onClick={this.loadTodoPage}
+              >
+                <ListItemIcon>
+                  <DealIcon
+                    className={this.isSelected("deals") && classes.selectedIcon}
+                  />
+                </ListItemIcon>
+                <ListItemText primary="Deals" />
+              </ListItem>
+              <ListItem
+                button
+                className={this.isSelected("surplus") && classes.selected}
+                key="Surplus"
+                onClick={this.loadSurplusPage}
+              >
                 <ListItemIcon>
-                  <AccountBoxIcon />
+                  <ChartIcon
+                    className={
+                      this.isSelected("surplus") && classes.selectedIcon
+                    }
```

Comment:
i think this should fit on one line

Reply:
same goes for the other instances where the className was broken into several lines. not sure if this is prettier helping or copy/paste

Reply:
all the weird line breaks are from prettier